component/buttonListeners/cam-listener.js

```diff
@@ -3,7 +3,7 @@ var { addEvent } = require('../../utils')
 const URL = require('url');
 
 class CamListener{
-    constructor(notificationService, doorbellId, db, camera, storage){
+    constructor(notificationService, doorbellId, db, camera, storage, button){
```

Comment:
Het is een beetje jammer dat CamListener nu afhankelijk wordt van button. Ik zal eens denken hoe we dat anders kunnen doen.

Reply:
Ja dit was voor zover ik kon bedenken met mijn onervaren node-hoofd de enige optie om de camListener de status van de button te laten weten... :)